RELEASE-LEADS.md

```diff
@@ -119,104 +109,27 @@ Make a copy of the
- # float forward in this repository.
- FLOATING_DEPS=(
--  "knative.dev/test-infra@${VERSION}"
--  "knative.dev/pkg@${VERSION}"
--  "knative.dev/caching@${VERSION}"
--  "knative.dev/networking@${VERSION}"
-+  "knative.dev/test-infra@release-x.y"
-+  "knative.dev/pkg@release-x.y"
-+  "knative.dev/caching@release-x.y"
-+  "knative.dev/networking@release-x.y"
- )
-```
+- [knative/hack](https://github.com/knative/hack)
 
-The downstream repositories this needs to happen on are:
+After `hack`:
 
-- [knative/client](https://github.com/knative/client)
+- [knative/pkg](https://github.com/knative/pkg)
+- [knative/test-infra](https://github.com/knative/test-infra)
 
-- [knative/operator](https://github.com/knative/operator)
+After `pkg`:
 
-- [knative/serving](https://github.com/knative/serving)
-- [knative-sandbox/net-certmanager](https://github.com/knative-sandbox/net-certmanager)
-- [knative-sandbox/net-contour](https://github.com/knative-sandbox/net-contour)
-- [knative-sandbox/net-http01](https://github.com/knative-sandbox/net-http01)
-- [knative-sandbox/net-istio](https://github.com/knative-sandbox/net-istio)
-- [knative-sandbox/net-kourier](https://github.com/knative-sandbox/net-kourier)
-
-- [knative/eventing](https://github.com/knative/eventing)
-- [knative/eventing-contrib](https://github.com/knative/eventing-contrib)
-- [knative-sandbox/eventing-kafka-broker](https://github.com/knative-sandbox/eventing-kafka-broker)
-- [knative-sandbox/discovery](https://github.com/knative-sandbox/discovery)
-- [knative-sandbox/eventing-autoscaler-keda](https://github.com/knative-sandbox/eventing-autoscaler-keda)
-- [knative-sandbox/eventing-awssqs](https://github.com/knative-sandbox/eventing-awssqs)
-- [knative-sandbox/eventing-camel](https://github.com/knative-sandbox/eventing-camel)
-- [knative-sandbox/eventing-ceph](https://github.com/knative-sandbox/eventing-ceph)
-- [knative-sandbox/eventing-couchdb](https://github.com/knative-sandbox/eventing-couchdb)
-- [knative-sandbox/eventing-github](https://github.com/knative-sandbox/eventing-github)
-- [knative-sandbox/eventing-gitlab](https://github.com/knative-sandbox/eventing-gitlab)
-- [knative-sandbox/eventing-kafka](https://github.com/knative-sandbox/eventing-kafka)
-- [knative-sandbox/eventing-natss](https://github.com/knative-sandbox/eventing-natss)
-- [knative-sandbox/eventing-prometheus](https://github.com/knative-sandbox/eventing-prometheus)
-- [knative-sandbox/eventing-rabbitmq](https://github.com/knative-sandbox/eventing-rabbitmq)
+- [knative/networking](https://github.com/knative/networking)
+- [knative/caching](https://github.com/knative/caching)
 
-Apply the changes the the **master branches**, run
-`hack/update-deps.sh --upgrade` (and potentially `hack/update-codegen.sh` if
-necessary) and PR the changes to the **master branch**. Don't cut the release
-branch yet.
+Automation will propagate these updates to all the downstream repos in the next
+few cycles.
```

Comment:
I think that we should note that (barring CPs) Serving/Eventing should be "releasable" by the end of this week.

Reply:
good point